README.md

```diff
@@ -88,7 +89,7 @@ All question types support the following CSV headers:
     -   Multiple Choice
     -   Upload
 -   `KEYWORD` (Recommended): One or more phrases that describe the question; separate each phrase with a comma
--   `SUBJECT` (Recommended): One or more subjects that are topical for this question; separate each phrase with a comma
+-   `SUBJECT` (Recommended): One or more subjects that are topical for this question; separate each phrase with a comma; subjects must match the predefined subjects in the database
```

Comment:
Looks good!  I think one thing we probably want to address in the future is how to add new subjects, maybe even reference the subjects.yaml file or something.

Reply:
Agreed. This is a little hand wavy for the moment. If we continue to use the rake task we'll want to include that command as well.